ga4gh/configtest.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import argparse
+import unittest
+import flask
+import types
+import os
+
+
+class TestConfigExists(unittest.TestCase):
+    """This class implements configuration checks for the GA4GH server. It is
+    structured as a set of unit tests, and relies on the unittest module.
+    However, as some of the tests require state injection (through command
+    line arguments) we need to break the usual unittest pattern and instead
+    implement our own simple test runner.
+    """
+
+    def __init__(self, parser=None):
```

Comment:
Following on from the point above, how about we pass configFile and configClass as arguments to this constructor, and take the command line argument parsing bit out of the setUp function below?